app/atomic_svc.py

```diff
@@ -0,0 +1,240 @@
+import json
+import glob
+import hashlib
+import os
+import re
+import shutil
+import yaml
+
+from collections import defaultdict
+from subprocess import DEVNULL, STDOUT, check_call
+
+from app.utility.base_world import BaseWorld
+from app.utility.base_service import BaseService
+
+PLATFORMS = dict(windows='windows', macos='darwin', linux='linux')
+EXECUTORS = dict(command_prompt='cmd', sh='sh', powershell='psh', bash='sh')
+RE_VARIABLE = re.compile('(#{(.*?)})', re.DOTALL)
+
+
+class AtomicService(BaseService):
+    def __init__(self, services):
+        self.log = self.add_service('atomic_svc', self)
+
+        # Atomic Red Team attacks don't come with the corresponding tactic (phase name)
+        # so we need to create a match between techniques and tactics.
+        # This variable is filled by self._populate_dict_techniques_tactics()
+        self.technique_to_tactics = None
+
+        self.atomic_dir = os.path.join('plugins', 'atomic')
+        self.repo_dir = os.path.join(self.atomic_dir, 'atomic-red-team')
+        self.data_dir = os.path.join(self.atomic_dir, 'data')
+        self.payloads_dir = os.path.join(self.atomic_dir, 'payloads')
+
+        # abilities ingested are counted in term of platform blocks created
```

Comment:
this comment doesn't add any value-- remove

Reply:
removed in 5b66cb5